roles/openshift_node/library/oc_csr_approve.py

```diff
@@ -268,7 +269,7 @@ def run(self):
         self.module.exit_json(**self.result)
 
 
-def run_module():
```

Comment:
Is there an advantage to using *args here instead of the module_args method below?

Reply:
We certainly could; the only difference I saw was that the module_args appeared to be an ansible expected type of nested dicts. *args expands to a list and **kwargs a dictionary. The run() function had no ties to ansible so I figured to provide in a simple format from the python args that were read/parsed as command line options. But technically there is no advantage one way or another

Reply:
I think this needs to be switched to module_args. This enables the parameter to be configured from the playbook where the script is used. For example, the kubeconfig and nodename parameters are defined below (lines 276,277) and used in the playbook:

https://github.com/openshift/openshift-ansible/blob/ce29a2ce7a45ef19d39c526c6d67e59b71a0756f/roles/openshift_node/tasks/config.yml#L200-L204

Reply:
Ok that way parameters make it from playbooks. I will make that change.

Reply:
I think that fix applied is what you wanted @jstuever.